deployments/charts/kuma/crds/kuma.io_meshtrusts.yaml

```diff
@@ -71,8 +71,11 @@ spec:
                 minItems: 1
                 type: array
               origin:
-                description: Origin specifies whether the resource was created from
-                  a MeshIdentity.
+                description: |-
```

Comment:
this is actually experimental and noone uses it, we can just break compatibility